modules/exploits/multi/http/jenkins_script_console.rb

```diff
@@ -40,7 +40,7 @@ def initialize(info = {})
               'CmdStagerFlavor' => [ 'certutil', 'vbs' ]
             }
           ],
-          ['Linux', { 'Arch' => ARCH_X86, 'Platform' => 'linux' }],
+          ['Linux', { 'Arch' => [ ARCH_X64, ARCH_X86 ], 'Platform' => 'linux' }],
```

Comment:
Just a confirmation question - when the user sets the target to linux, as they expected to manually change the payload too? 👀

Reply:
Yeah but I think if the payload is left unset, the default payload selection logic will kick in and pick one automagically. Would the alternative be to hard code a default here?

Reply:
For me a payload isn't rechoosen; Potentially due to https://github.com/rapid7/metasploit-framework/pull/13566